packages/cli/src/commands/init.ts

```diff
@@ -86,7 +86,7 @@ export const handler = buildHandler(
           try {
             const url = `${getMailingAPIBaseURL()}/api/newsletterSubscribers`;
 
-            fetch(url, {
+            await fetch(url, {
```

Comment:
here we're purposefully not awaiting bc we don't care about the response and don't want to make the user wait. Is that what `void` is for?

Reply:
Yeah, i will change it to void.  If we can't get the linting to work, we could just pull out the changes that it turned up and do that for now